circle.yml

```diff
@@ -249,6 +247,24 @@ jobs:
           paths:
             - ~/build/corpus
 
+  spectest:
+    executor: linux-gcc-9
+    steps:
+      - run:
+          name: "Download spectest files"
+          command: |
+            git clone https://github.com/wasmx/wasm-spec --branch vanilla-json --single-branch
```

Comment:
```suggestion
            git clone https://github.com/wasmx/wasm-spec --branch vanilla-json --single-branch --depth 1
```

Reply:
Manual says it also implies `--single-branch`. Should remove that then?